catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -108,13 +116,31 @@ public void setRegistryDisabled(boolean registryDisabled) {
         return this.registryBypassPolicyStrings;
     }
 
-    public List<String> getWriteAccessPolicyStrings() {
-        return writeAccessPolicyStrings;
+    public List<String> getCreateAccessPolicyStrings() {
+        return createAccessPolicyStrings;
     }
 
-    public void setWriteAccessPolicyStrings(List<String> writeAccessPolicyStrings) {
-        this.writeAccessPolicyStrings = writeAccessPolicyStrings;
-        parsePermissionsFromString(writeAccessPolicyStrings, writeAccessPolicy);
+    public void setCreateAccessPolicyStrings(List<String> accessPolicyStrings) {
+        this.createAccessPolicyStrings = accessPolicyStrings;
+        parsePermissionsFromString(accessPolicyStrings, createAccessPolicy);
+    }
+
+    public void setUpdateAccessPolicyStrings(List<String> accessPolicyStrings) {
+        this.updateAccessPolicyStrings = accessPolicyStrings;
+        parsePermissionsFromString(accessPolicyStrings, updateAccessPolicy);
+    }
+
+    public List<String> getUpdateAccessPolicyStrings() {
```

Comment:
And do we care that we're returning collections that can be modified by the caller?